test/Chainweb/Test/Pact/TransactionTests.hs

```diff
@@ -343,13 +332,25 @@ testUpgradeScript
     :: FilePath
     -> V.ChainId
     -> BlockHeight
-    -> (Either SomeException (T2 (CommandResult [TxLog Value]) (Maybe ModuleCache)) -> IO ())
+    -> (T2 (CommandResult [TxLog Value]) (Maybe ModuleCache) -> IO ())
     -> IO ()
-testUpgradeScript script cid bh test = do
+testUpgradeScript fp cid bh f
+    = testUpgradeScript' fp v cid bh f
+
+testUpgradeScript'
```

Comment:
It looks like this is never directly called, do you want to leave this version in?

Reply:
Yeah, i could probably remove this. I intended it to allow us to call scripts with different versions, but we only ended up testing dev here.